Tests/CustomEntitySearchTest/TestData.cs

```diff
@@ -0,0 +1,274 @@
+﻿using AzureCognitiveSearch.PowerSkills.Common;
+using AzureCognitiveSearch.PowerSkills.Text.CustomEntitySearch;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace AzureCognitiveSearch.PowerSkills.Tests.CustomEntitySearchTest
+{
+    public class TestData
+    {
+        public const string hostAddress = "http://localhost:7071/api/CustomEntitySearch";
+        public const string missingWordsBadRequestInput = @"{
+    ""values"": [
+        {
+            ""recordId"": ""1"",
+            ""data"":
+            {
+                ""text"":  ""Pablo Picasso"",
+                ""languageCode"": ""en""
+            }
```

Comment:
nit: remove languageCode for now since its not used

Reply:
same for missingTextBadRequestInput below

---
In reply to: [307376004](https://github.com/Azure-Samples/azure-search-power-skills/pull/1#discussion_r307376004) [](ancestors = 307376004)